extide/gradle/src/org/netbeans/modules/gradle/api/NbGradleProject.java

```diff
@@ -307,7 +308,18 @@ public void removePropertyChangeListener(PropertyChangeListener propertyChangeLi
     public static NbGradleProject get(Project project) {
         return project instanceof NbGradleProjectImpl ? ((NbGradleProjectImpl) project).getProjectWatcher() : null;
     }
-
+    
+    /**
+     * Returns accessor for Gradle project files. Note that the returned instance is immutable, possibly lazy-initialized.
+     * A change (creation, removal) to project files will not be reflected by the {@link GradleFiles} instance, but this method
+     * may return a new instance.
+     * @return files accessor.
+     * @since 2.24
+     */
+    public GradleFiles getFiles() {
```

Comment:
Could the naming of the method be consistent and use getGradleFiles instead of getFiles?